main.cpp

```diff
@@ -1010,9 +891,273 @@ Part 1e - Step 19: Request a review
 paste your code below
 */
 
+struct Train
+{
+    //number of train cars
+    int numTrainCars = 5;
+    //number of seats
+    int numSeats = 100;
+    //the amount of electrcity used per week
+    float electricityUsedPerWeek = 28000.f; 
+    //number of miles traveled per day
+    double numMilesTravelPerDay = 268.6;
+    //ticket price
+    double ticketPrice = 12;
+
+    //3 things it can do:
+    //take customer to destination
+    void takeCustomerToDestination(bool isCustomerDestination);
+
+    //check ticket
+    void checkTicket(bool customerBoughtTicket);
+
+    //pickup customers at station
+    bool pickupCustomer(int amountCustomerAtStop); //returns true if there is any customers to pick up
+};
+
+struct Restaurant 
+{
+    //number of food items
+    int numFoodItems = 15;
+    //number of plates
+    int numPlates = 200;
+    //number of chairs
+    int numChairs = 50;
+    //number of tables
+    int numTables = 15;
+    //operating hours
+    double operatingHours = 9.5;
+
+    //3 things it can do:
+    //feed customer
+    void feedCustomer(bool customerIsHungry, int numCustomerAtTable, bool foodOrderedAvailable);
+
+    //serve customer
+    void serveCustomer(bool customerOrderedFood);
+
+    //reserve seats for customer
+    bool reserveSeatsCustomer(int numCustomer, bool fullyBooked); //returns true is the amount of customers wanting to book can book
+};
+
+struct Airport
+{
+    //number of planes
+    int numPlanes = 30;
+    //lenght of runway in m
+    int lengthRunwayM = 2800;
+    //number of gates
+    int numGates = 20;
+    //secirity line wait time
+    int securityLineWaitTime = 22;
+    //number of private jets
+    int numPrivateJets = 2;
+    
+    struct CommercialPlane
+    {
+        bool isAPrivateJet = false;
+        int numEngines = 4;
+        std::string manufacturer = "Boeing";
+        std::string model = "747";
+        int numSeats = 416;
+
+        void flightCheck(float milesTraveled, int yearsOld, bool warningLightOn);
```

Comment:
`flight check` is a noun.  Function names reflect actions.
The VERB should always be the first word in the function name.
- [ ] revise